libflux/core/src/formatter/mod.rs

```diff
@@ -652,6 +653,43 @@ impl Formatter {
         self.write_rune(']')
     }
 
+    fn format_dict_expression(&mut self, n: &ast::DictExpr) {
+        let multiline = n.elements.len() > 4 || n.base.is_multiline();
+        self.format_comments(&n.lbrack);
+        self.write_rune('[');
+        if multiline {
+            self.temp_tabstops = true;
```

Comment:
Why is temp_tabstops being used here? As I understood it that flag is only used when a comment interrupts a line.

Reply:
I didn't gather that when I was reviewing the code. It seems like it's only used when formatting records as far as I could tell. I added it to be consistent with how arrays are formatted.

Reply:
ah yes I think I have it confused with `temp_indent`. Nevermind this looks good here.